tests/test_database_types.py

```diff
@@ -356,33 +359,90 @@ def expand_params(testcase_func, param_num, param):
     return name
 
 
-def _insert_to_table(conn, table, values):
-    insertion_query = f"INSERT INTO {table} (id, col) "
+def _insert_to_table(conn, table, values, type):
+    current_n_rows = conn.query(f"SELECT COUNT(*) FROM {table}", int)
+    if current_n_rows == N_SAMPLES:
+        assert BENCHMARK, "Table should've been deleted, or we should be in BENCHMARK mode"
+        return
+    elif current_n_rows > 0:
+        logging.error(f"{conn.name}: Table {table} has {current_n_rows}, expected {N_SAMPLES} or 0. Recreating table")
```

Comment:
Maybe this should be a warning?

Reply:
Default logging error level is ERROR to have less noise for test output.

Reply:
Maybe I'm missing something, but it should be simple to set the test logger to warning, and the data-diff loggers to error, no?

Reply:
Yeah, that was what I was getting at in my last setntence.. I will just remove tit then